tests/unit/conftest.py

```diff
@@ -16,58 +18,43 @@ def list_dpkg():
-                    "display-name": "Canonical",
-                    "validation": "verified",
-                },
-                "developer": "canonical",
-                "status": "active",
-                "type": "app",
-                "base": "core18",
-                "version": "2.5.0",
-                "channel": "stable",
-                "tracking-channel": "latest/stable",
-                "ignore-validation": False,
-                "revision": "281",
-                "confinement": "strict",
-                "private": False,
-                "devmode": False,
-                "jailmode": False,
-                "apps": [
-                    {"snap": "cvescan", "name": "cvescan"},
-                    {"snap": "cvescan", "name": "sh"},
-                ],
-                "license": "GPL-3.0",
-                "mounted-from": "/var/lib/snapd/snaps/cvescan_281.snap",
-                "links": {
-                    "contact": ["https://github.com/canonical/sec-cvescan/issues"],
-                    "website": ["https://github.com/canonical/sec-cvescan"],
+    """Equivalent of the content of the file at /var/lib/snapd/state.json ."""
+    snaps = {
+        "data": {
+            "snaps": {
+                "core20": {
+                    "type": "base",
+                    "sequence": [
+                        {
+                            "name": "core20",
+                            "snap-id": "DLqre5XGLbDqg9jPtiAhRRjDuPVa5X1q",
+                            "revision": "1822",
+                            "channel": "latest/stable",
+                            "links": {"contact": ["https://github.com/snapcore/core20/issues"]},
+                            "contact": "https://github.com/snapcore/core20/issues",
+                            "title": "core20",
+                            "summary": "Runtime environment based on Ubuntu 20.04",
+                            "description": "The base snap based on the Ubuntu 20.04 release.",
+                        },
+                        {
+                            "name": "core20",
+                            "snap-id": "DLqre5XGLbDqg9jPtiAhRRjDuPVa5X1q",
+                            "revision": "1828",
+                            "channel": "latest/stable",
+                            "links": {"contact": ["https://github.com/snapcore/core20/issues"]},
+                            "contact": "https://github.com/snapcore/core20/issues",
+                            "title": "core20",
+                            "summary": "Runtime environment based on Ubuntu 20.04",
+                            "description": "The base snap based on the Ubuntu 20.04 release.",
+                        },
+                    ],
+                    "active": True,
+                    "current": "1828",
+                    "channel": "latest/stable",
+                    "last-refresh-time": "2023-03-08T09:06:07.732371956-03:00",
+                    "version": "20230207",
```

Comment:
Since you are trying to create a equivalent of the content in `/var/lib/snapd/state.json`, I think we should remove the `version` key-value pair here.